Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.jquery.ts

```diff
@@ -1031,17 +1076,17 @@
 
             list = $("<ul/>")
             .appendTo(innerDiv)
-            .mouseover(function (event) {
-                if (target(event).nodeName && target(event).nodeName.toUpperCase() === 'LI')
+            .mouseover(function (event: JQueryMouseEventObject) {
+                if ((target(event) as Node).nodeName && (target(event) as Node).nodeName.toUpperCase() === 'LI')
                 {
-                  active = $("li", list).removeClass(CLASSES.ACTIVE).index(target(event));
-                  var activeItem = $(target (event));
+                  active = $("li", list).removeClass(CLASSES.ACTIVE).index(target(event) as JQuery);
+                  var activeItem = $(target (event) as JQuery);
```

Comment:
why do we have to JQuery-cast the value we want to pass into the JQuery 'constructor'?